[2025-03-17T22:30:26.808+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Google_map.transform_data_phase_1_task manual__2025-03-15T15:55:06.054403+00:00 [queued]>
[2025-03-17T22:30:27.034+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Google_map.transform_data_phase_1_task manual__2025-03-15T15:55:06.054403+00:00 [queued]>
[2025-03-17T22:30:27.035+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-03-17T22:30:27.900+0000] {taskinstance.py:2191} INFO - Executing <Task(BashOperator): transform_data_phase_1_task> on 2025-03-15 15:55:06.054403+00:00
[2025-03-17T22:30:27.968+0000] {standard_task_runner.py:60} INFO - Started process 15989 to run task
[2025-03-17T22:30:28.080+0000] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'Google_map', 'transform_data_phase_1_task', 'manual__2025-03-15T15:55:06.054403+00:00', '--job-id', '198', '--raw', '--subdir', 'DAGS_FOLDER/google_map_dag_etl.py', '--cfg-path', '/tmp/tmp_e_hmxx0']
[2025-03-17T22:30:28.111+0000] {standard_task_runner.py:88} INFO - Job 198: Subtask transform_data_phase_1_task
[2025-03-17T22:30:29.918+0000] {task_command.py:423} INFO - Running <TaskInstance: Google_map.transform_data_phase_1_task manual__2025-03-15T15:55:06.054403+00:00 [running]> on host DESKTOP-AVAHALM.
[2025-03-17T22:30:32.982+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='master_m2si' AIRFLOW_CTX_DAG_ID='Google_map' AIRFLOW_CTX_TASK_ID='transform_data_phase_1_task' AIRFLOW_CTX_EXECUTION_DATE='2025-03-15T15:55:06.054403+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-15T15:55:06.054403+00:00'
[2025-03-17T22:30:32.986+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-03-17T22:30:32.987+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'cd ~/data_warehouse_project && dbt run --profiles-dir ~/.dbt --models cleaned_reviews']
[2025-03-17T22:30:33.090+0000] {subprocess.py:86} INFO - Output:
[2025-03-17T22:30:54.116+0000] {subprocess.py:93} INFO - [0m22:30:54  Running with dbt=1.9.0-b2
[2025-03-17T22:30:58.820+0000] {subprocess.py:93} INFO - [0m22:30:58  Registered adapter: postgres=1.8.2
[2025-03-17T22:31:03.096+0000] {subprocess.py:93} INFO - [0m22:31:03  [[33mWARNING[0m]: Configuration paths exist in your dbt_project.yml file which do not apply to any resources.
[2025-03-17T22:31:03.096+0000] {subprocess.py:93} INFO - There are 1 unused configuration paths:
[2025-03-17T22:31:03.097+0000] {subprocess.py:93} INFO - - models.data_warehouse_project.example
[2025-03-17T22:31:07.055+0000] {subprocess.py:93} INFO - [0m22:31:07  Found 6 models, 1 source, 424 macros
[2025-03-17T22:31:07.125+0000] {subprocess.py:93} INFO - [0m22:31:07
[2025-03-17T22:31:07.128+0000] {subprocess.py:93} INFO - [0m22:31:07  Concurrency: 4 threads (target='dev')
[2025-03-17T22:31:07.144+0000] {subprocess.py:93} INFO - [0m22:31:07
[2025-03-17T22:31:13.600+0000] {subprocess.py:93} INFO - [0m22:31:13  1 of 1 START sql table model public.cleaned_reviews ............................ [RUN]
[2025-03-17T22:31:15.521+0000] {subprocess.py:93} INFO - [0m22:31:15  1 of 1 OK created sql table model public.cleaned_reviews ....................... [[32mSELECT 7[0m in 1.90s]
[2025-03-17T22:31:16.314+0000] {subprocess.py:93} INFO - [0m22:31:16
[2025-03-17T22:31:16.331+0000] {subprocess.py:93} INFO - [0m22:31:16  Finished running 1 table model in 0 hours 0 minutes and 9.15 seconds (9.15s).
[2025-03-17T22:31:17.114+0000] {subprocess.py:93} INFO - [0m22:31:17
[2025-03-17T22:31:17.124+0000] {subprocess.py:93} INFO - [0m22:31:17  [32mCompleted successfully[0m
[2025-03-17T22:31:17.126+0000] {subprocess.py:93} INFO - [0m22:31:17
[2025-03-17T22:31:17.131+0000] {subprocess.py:93} INFO - [0m22:31:17  Done. PASS=1 WARN=0 ERROR=0 SKIP=0 TOTAL=1
[2025-03-17T22:31:21.035+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-03-17T22:31:22.588+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=Google_map, task_id=transform_data_phase_1_task, execution_date=20250315T155506, start_date=20250317T223026, end_date=20250317T223122
[2025-03-17T22:31:22.941+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-03-17T22:31:23.849+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
